cbv3_django_prototype/cbv3_django_prototype/resources/tests.py

```diff
@@ -10,14 +10,18 @@
 class ResourcesTests(APITestCase):
 
     def setUp(self):
-        """
-        Loads users.json and resources.json fixtures into test DB
-        """
+
         call_command('loaddata', 'users.json', verbosity=0)
         call_command('loaddata', 'resources.json', verbosity=0)
+        call_command('loaddata', 'tags.json', verbosity=0)
+
         model = get_user_model()
-        self.person = model.objects.create_user(username='PetuniaPig', email='pretty.piglet@pigfarm.org',
-                                           password='codebuddies', is_superuser=True)
+        self.person = model.objects.create_user(
```

Comment:
Great question.  So the `users.json` file is loaded **directly into**  the db before testing ever starts.  It's using SQL, and I don't think it is really interacting a whole lot with the ORM.  

During the test, calling `create` mimics some of the actions of having a brand-new user register and get created in the DB, so it is calling  ORM code.  

Then it turns around and asks for a token.  It may be overkill, since the tests for the `auth` endpoint also attempt to cover this.  I will remove the create from this test and use a user from `users.json`, since these test are more about  `resources` than `users`.  Additionally, when the tests are re-written for `factory-boy` or `model-mommy`, the process of creating test data will move away from fixtures.